.github/workflows/actions.yml

```
@@ -0,0 +1,90 @@
+name: quicknxs_ci
+
+env:
+  TRIGGER_URL: "https://code.ornl.gov/api/v4/projects/11852/trigger/pipeline"
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches: [next]
+    tags: ['v*']
+
+jobs:
+
+  linux:
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash -l {0}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04]
+        python-version: ["3.8"]
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+          mamba-version: "*"
+          environment-file: environment.yml
+      - name: Unit test with code coverage
+        run: xvfb-run -a python -m pytest -vv --cov=reflectivity_ui --cov-report=xml --cov-report=term
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v3
+        with:
+          verbose: true
+      - name: Build Wheel
+        run: |
+          python -m build --wheel --no-isolation
+          check-wheel-contents dist/reflectivity_ui-*.whl
+
+      - name: Build Conda package
+        run: |
+          conda build --output-folder . . -c mantid
+          conda verify noarch/reflectivity_ui-*.tar.bz2
+      - name: Deploy to Anaconda
+        if: startsWith(github.ref, 'refs/tags/v')
+        env:
+          ANACONDA_API_TOKEN: ${{ secrets.ANACONDA_TOKEN }}
+          IS_RC: ${{ contains(github.ref, 'rc') }}
+          IS_DEV: ${{ github.ref == 'refs/heads/next' }}
+        run: |
+          conda install -y anaconda-client
+          # label is main or rc depending on the tag-name
+          CONDA_LABEL="main"
+          if [ "${IS_RC}" = "true" ]; then CONDA_LABEL="rc"; fi
+          if [ "${IS_DEV}" = "true" ]; then CONDA_LABEL="dev"; fi
+          echo pushing ${{ github.ref }} with label $CONDA_LABEL
+          anaconda upload --label $CONDA_LABEL conda.recipe/noarch/reflectivity_ui-*.tar.bz2
+
+  trigger-deploy:
+    runs-on: ubuntu-22.04
+    needs: [linux]
+    # only trigger deploys from protected branches
+    if: ${{ github.ref_protected || github.ref_type == 'tag' }}
+    steps:
+      - name: Determine Environment
+        uses: neutrons/branch-mapper@main
+        id: conda_env_name
+        with:
+          prefix: quicknxs
```

```suggestion
          prefix: quicknxs2
```

I think this needs to be `quicknxs2` to match the `.yml` files in quicknxs-deploy.